packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -0,0 +1,386 @@
+                                <NavigationMenu
+                                    menu={menu}
+                                    expanded={true}
+                                    onLocationUpdate={this.handleUpdateRoute}
+                                />
+                            </Pane>
+                            <Canvas jssStyleSheet={this.canvasStyleOverrides}>
+                                <Row fill={true}>
+                                    <div style={{ width: "100%" }}>
+                                        <div style={{ padding: "7px 10px" }}>
+                                            {this.renderScenarioSelect()}
+                                            {this.renderDirectionToggle()}
+                                        </div>
+                                        <Viewer
+                                            iframeSrc={"/preview"}
+                                            iframePostMessage={this.state.viewConfig}
+                                            width={this.state.width}
+                                            height={this.state.height}
+                                            onUpdateHeight={this.handleUpdateHeight}
+                                            onUpdateWidth={this.handleUpdateWidth}
+                                            viewerContentProps={this.state.scenario}
+                                            responsive={true}
+                                            jssStyleSheet={this.viewerStyleOverrides}
+                                        />
+                                    </div>
+                                </Row>
+                                <Row
+                                    resizable={true}
+                                    resizeFrom={RowResizeDirection.north}
+                                    initialHeight={400}
+                                >
+                                    <Background
+                                        value={this.backgrounds.L4}
+                                        style={{ width: "100%" }}
+                                    >
+                                        <Pivot
+                                            label={"documentation"}
+                                            items={this.renderPivotItems()}
+                                            jssStyleSheet={{
+                                                pivot: {
+                                                    height: "100%",
+                                                },
+                                                pivot_tabPanels: {
+                                                    overflow: "auto",
+                                                    padding: "0 12px",
+                                                    height: "calc(100% - 32px)",
+                                                },
+                                            }}
```

Comment:
you shouldn't create objects inline when doing `jssStyleSheet` overrides because the JSSManager will see a new stylesheet every render. You should store this as a variable or as a member on the class

Reply:
Thanks for the catch! Looks like I missed this one.